pages/dkp/kommander/1.4/install-airgapped/index.md

```diff
@@ -96,6 +96,8 @@ Your `cluster.yaml` file should look similar to the following for Kommander Addo
 
  If you wish to disable the Docker registry certificate verification, set `docker-registry-insecure-skip-tls-verify` to `true` in the `utilityApiserver`'s `extraArgs`. We encourage you to keep the certificate verification enabled to validate all TLS connections to the registry.
 
+ This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0][https://www.gnu.org/licenses/agpl-3.0.en.html]. The complete source code for MinIO is available for the [2021 release][[https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z] of Kommander and the [2022 release][https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z] of Kommander.
```

Comment:
hey @mwrightd2 these messages need to match what's in this folder verbatim. 

https://drive.google.com/drive/u/0/folders/1eb6tnAx6Il9YvyVKZ2Oz0Scubhuwis7P